lib/alice.dart

```diff
@@ -15,28 +18,20 @@ class Alice {
   AliceCore _aliceCore;
   AliceHttpClientAdapter _httpClientAdapter;
 
-  Alice._(
-    this.darkTheme,
-    this._navigatorKey,
-    this._aliceCore,
-    this._httpClientAdapter,
-  );
+  Alice._(this.darkTheme, this._navigatorKey, this._aliceCore,
+      this._httpClientAdapter);
 
   /// Creates alice instance.
   factory Alice({
     GlobalKey<NavigatorState> navigatorKey =
         const GlobalObjectKey<NavigatorState>('AliceNavigatorState'),
     bool darkTheme = false,
+    AliceCustomColors customColors = const AliceCustomColors(),
```

Comment:
let's add a comment here with small description and usage example as well.

Reply:
done